<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WsfDrawDockWidget</class>
 <widget class="QDockWidget" name="WsfDrawDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>321</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>WsfDraw</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="networkCheckBox">
      <property name="text">
       <string>Show Draws From Networked Simulations</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="showAllCheckBox">
      <property name="text">
       <string>Show All</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
